openehr-rm/src/test/java/com/nedap/archie/rm/support/identification/VersionTreeIdTest.java

```diff
@@ -0,0 +1,82 @@
+package com.nedap.archie.rm.support.identification;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class VersionTreeIdTest {
+
+    @Test
+    public void isBranch() {
+        VersionTreeId version = new VersionTreeId("1");
+        assertEquals(Boolean.FALSE, version.isBranch());
+
+        version = new VersionTreeId("1.1.1");
+        assertEquals(Boolean.TRUE, version.isBranch());
+
+        version = new VersionTreeId("123.123.123");
+        assertEquals(Boolean.TRUE, version.isBranch());
+
+        version = new VersionTreeId("2.0.0");
+        assertEquals(Boolean.TRUE, version.isBranch());
+
+        // invalid entries
+        version = new VersionTreeId("0.0.1");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+
+        version = new VersionTreeId("1.");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+
+        version = new VersionTreeId("1.1");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+
+        version = new VersionTreeId("1.1.");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+
+        version = new VersionTreeId(".1.");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+
+        version = new VersionTreeId("..1");
+        assertThrows(IllegalArgumentException.class, version::isBranch);
+    }
+
+    @Test
+    public void getTrunkVersion() {
+        VersionTreeId version = new VersionTreeId("1");
+        assertEquals("1", version.getTrunkVersion());
+
+        version = new VersionTreeId("43");
+        assertEquals("43", version.getTrunkVersion());
+
+        version = new VersionTreeId("1.1.1");
+        assertEquals("1", version.getTrunkVersion());
+
+        version = new VersionTreeId("123.123.123");
+        assertEquals("123", version.getTrunkVersion());
```

Comment:
As all three version parts are equal, this test doesn't actually test if the correct part is returned. Can you change this test to use three different parts, e.g. `1.2.3`?

Reply:
Good catch, thanks! Seems I forgot to do this here.